text/src/autogluon/text/automm/data/process_text.py

```diff
@@ -35,6 +41,59 @@
 }
 
 
+def construct_augmenter(
```

Comment:
I think making this a static method inside `TextProcessor` is better. There are three reasons:

1. This is mainly used by `TextProcessor`. If it aims to be a general util, then we need to move it to `data/utils.py`.
2. To be be consistent with other processors. Static methods also exist in other processors. Each processor file has one processor class with its class methods and closely related static methods.
3. The name `construct_augmenter` sounds like a general name, but it is to construct text augmenter. So, it is better to put inside `TextProcessor`.

Reply:
I think it's a generic function and can be useful elsewhere. Thus, staticmethod may not be a good design.

Reply:
A static method doesn't mean it can't be used in other places. It is mainly to organize the methods based on their semantic meanings. Obviously, this method is mainly tied to this class.

Reply:
After offline discussion with @sxjscience, I think it's fine to keep this function here. But we need to change the name to make it specific for text.